### ISSUE TITLE
Allow disabling autoupdate in `pre-commit.ci` GitHub app

### ISSUE THREAD

**Issue by blakeNaccarato** [pre-commit/pre-commit#2670]
I use [Renovate](https://www.mend.io/free-developer-tools/renovate/) to bump my pre-commit config dependencies in a more granular fashion than the built-in autoupdate feature. Please consider introducing an `autoupdate_schedule: never` option in the configuration so that I can use all the other nice features of your CI app without worrying about it creating noisy update PRs. Currently I have it set to `quarterly` and just ignore/close the PRs.

**Comment by asottile** [comment]
the issue tracker for pre-commit.ci is at https://github.com/pre-commit-ci/issues -- where you'll find several duplicates addressing your inquiry.  please read the template next time which prompts you to go there

**Comment by blakeNaccarato** [comment]
I see. Since I was making a feature request, I skipped the "bug report" issue template altogether. I also checked the `pre-commit` GitHub org and didn't see a `pre-commit-ci` repo, nor does https://pre-commit.ci/ have a link to the appropriate repository for bug reports, so I figured issues were to be raised here.

I am fairly diligent when it comes to searching before posting issues, but I guess I missed the link this time. Perhaps there could be some minor tweaks to increase discoverability to users, either over at https://pre-commit.ci/ or on the "New Issue" page prior to clicking through to "bug report" issue template?
